### DISCUSSION TITLE
My business account was rejected on "Employment verification" step

### DISCUSSION THREAD

**Discussion by qiliangjia-cn** [microsoft/MicrosoftEdge-Extensions#174] [Q&A]
![350240065-14d4f87d-0cca-4652-b7c1-7a49a537a00d](https://github.com/user-attachments/assets/3d661446-0de9-464a-abac-a39e1dac6734)
我注册了公司类型的帐户，但在验证过程中，我在“就业验证”步骤中一直被拒绝！没有具体的错误提示，也没办法联系到后台工作人员！
卖家ID:88539100

**Comment by liambolling** [comment]
I can't get past this too...

**Comment by ManikanthMSFT** [comment]
Hi Dev's, thank you for reaching out to us. To ensure we can assist you effectively, please contact us at **ext_dev_support@microsoft.com** along with your Store ID. Our team is ready to provide the support you need.

**Comment by liambolling** [comment]
It's been a week and I received an email asking me to reverify (which is broken on the website in Edge and Chrome). This is still not working / nobody can verify their employment (why is this even needed?) Google doesn't make me do this to create a Chrome extension.

**Comment by liambolling** [comment]
Now this is closed?
